### ISSUE TITLE
Non-Steam games don't accept keyboard or mouse input via streaming

### ISSUE THREAD

**Issue by CaptSpify** [ValveSoftware/SteamOS#208]
I don't know if non-steam games are supposed to work, but...
when I add a game to steam (via Games > Add a non-steam game to my library), I am unable to stream them. I can launch them, but I can't send any input controls (mouse & keyboard).
If I control the games via the Windows-OS, I see the changes on Steam-OS.

Please let me know if any other information is needed.

**Comment by ProfessorKaos64** [comment]
Non-steam games, even added on your local SteamOS box are beholden to the technologies the games use for input control. Games that made use of SDL, such as SuperTuxKart, work fine, otherwise, you will need to employ some trickey/mapping with Antimicro or swap out profiles using xboxdrv. I tend to setup a profile in Antimicro.
